training/watch.md

```diff
@@ -0,0 +1,29 @@
+---
+---
+
+# 3. When do you watch spam?
+Adding websites and keywords to be watched by SmokeDetector is a big part of being a privileged
+user on Charcoal. Unless a post is ***obvious*** spam, it can sometimes be a bit overwhelming to
```

Comment:
Feels like this second sentence should start a new paragraph, or be its own bullet item (do some background/preliminary research on the meaning of the term before you add it).

Probably would also be good not to make it so software-development--specific, but that's a bit harder.

Reply:
Done.